TROUBLESHOOTING.md

```diff
@@ -25,3 +25,20 @@ Ensure that the Ollama URL is correctly formatted in the application settings. F
 It is crucial to include the `/api` at the end of the URL to ensure that the Ollama Web UI can communicate with the server.
 
 By following these troubleshooting steps, you should be able to identify and resolve connection issues with your Ollama server configuration. If you require further assistance or have additional questions, please don't hesitate to reach out or refer to our documentation for comprehensive guidance.
+
+## Running ollama-webui as a cintainer on Apple Silicon Mac
+
+If you are running Docker on a M{1..3} based Mac and have taken the steps to run an x86 container, add "--platform linux/amd64" to the docker run command.
```

Comment:
Thanks for the PR! I think it's also worth mentioning that even without the `--platform linux/amd64` flag it should work just as fine, it just outputs the warning in the terminal when you execute the command.